server/routes/templates.ts

```diff
@@ -148,6 +148,14 @@ templateRoutes.route("/").get(authenticateWithRedirect, async (request, response
 	else {
 		applyBranches = (await Branches.BranchConfig.loadAllBranches("Application") as Branches.ApplicationBranch[]);
 	}
+	let applyTimes = applyBranches.reduce((map, branch) => {
+		map[branch.name] = branch;
+		return map;
+	}, {} as IDeadlineMap);
```

Comment:
Can you move the `IDeadlineMap` and `IBranchOpenClose` interfaces up so they appear defined before being used?